docs/fsharp/using-fsharp-on-azure/blob-storage.md

```diff
@@ -106,66 +96,49 @@ For example, consider the following set of block blobs in a container named `pho
-
-```console
-Block blob of length 4: https://<accountname>.blob.core.windows.net/photos/2015/architecture/description.txt
-Block blob of length 314618: https://<accountname>.blob.core.windows.net/photos/2015/architecture/photo3.jpg
-Block blob of length 522713: https://<accountname>.blob.core.windows.net/photos/2015/architecture/photo4.jpg
-Block blob of length 4: https://<accountname>.blob.core.windows.net/photos/2016/architecture/description.txt
-Block blob of length 419048: https://<accountname>.blob.core.windows.net/photos/2016/architecture/photo5.jpg
-Block blob of length 506388: https://<accountname>.blob.core.windows.net/photos/2016/architecture/photo6.jpg
-Block blob of length 399751: https://<accountname>.blob.core.windows.net/photos/2016/photo7.jpg
-Block blob of length 505623: https://<accountname>.blob.core.windows.net/photos/photo1.jpg
-```
-
 ## Download blobs
 
-To download blobs, first retrieve a blob reference and then call the `DownloadToStream` method. The following example uses the `DownloadToStream` method to transfer the blob contents to a stream object that you can then persist to a local file.
+To download blobs, first retrieve a blob reference and then call the `DownloadTo` method. The following example uses the `DownloadTo` method to transfer the blob contents to a stream object that you can then persist to a local file.
 
-[!code-fsharp[BlobStorage](~/samples/snippets/fsharp/azure/blob-storage.fsx#L95-L101)]
+[!code-fsharp[BlobStorage](../../../samples/snippets/fsharp/azure/blob-storage.fsx#L71-L76)]
 
-You can also use the `DownloadToStream` method to download the contents of a blob as a text string.
+You can also use the `DownloadTo` method to download the contents of a blob as a text string.
 
-[!code-fsharp[BlobStorage](~/samples/snippets/fsharp/azure/blob-storage.fsx#L103-L106)]
+[!code-fsharp[BlobStorage](../../../samples/snippets/fsharp/azure/blob-storage.fsx#L78-L81)]
 
 ## Delete blobs
 
 To delete a blob, first get a blob reference and then call the
 `Delete` method on it.
 
-[!code-fsharp[BlobStorage](~/samples/snippets/fsharp/azure/blob-storage.fsx#L112-L116)]
+[!code-fsharp[BlobStorage](../../../samples/snippets/fsharp/azure/blob-storage.fsx#L87-L91)]
 
 ## List blobs in pages asynchronously
 
 If you are listing a large number of blobs, or you want to control the number of results you return in one listing operation, you can list blobs in pages of results. This example shows how to return results in pages asynchronously, so that execution is not blocked while waiting to return a large set of results.
 
-This example shows a flat blob listing, but you can also perform a hierarchical listing, by setting the `useFlatBlobListing` parameter of the `ListBlobsSegmentedAsync` method to `false`.
+This example shows a flat blob listing, but you can also perform a hierarchical listing, by using the `AsPages` method of the `BlobClient` .
```

Comment:
That's not really accurate.  There's a different method called `GetBlobsByHierarchy`.  `AsPages` is just for manually iterating through pages instead of having us do that for you behind the scenes.